src/main/java/de/tum/in/www1/artemis/web/rest/dto/PostContextFilter.java

```diff
@@ -22,11 +22,11 @@ public class PostContextFilter {
 
     private String searchText;
 
-    private Boolean filterToUnresolved;
+    private boolean filterToUnresolved = false;
 
-    private Boolean filterToOwn;
+    private boolean filterToOwn = false;
 
-    private Boolean filterToAnsweredOrReacted;
+    private boolean filterToAnsweredOrReacted = false;
```

Comment:
Explicitly assigning false as default value can be removed since these are `boolean` but it wouldn't hurt to be explicit as well.

Reply:
I usually prefer to explicitly set booleans to false for two reasons:
- It is clear that false is the intentional default value and that it even should have a default value
- That passing this attribute in the JSON is (still) entirely optional

Reply:
I personally would not do this explicit declaration but every developer has their own preferences. 👍🏻